prac_05/state_names.py

```diff
@@ -0,0 +1,20 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+# state_code = input("Enter short state: ")
```

Comment:
Is there a reason why you left this comment in the final copy?

Reply:
I do not have a reason, it was part of the starting code however wasn't the final result asked for. I just didn't know whether to remove it or not.